client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-item.js

```diff
@@ -113,11 +121,20 @@ class LabelItem extends Component {
 					{ label.showDetails && this.renderLabelDetails( label ) }
 				</p>
 				{ label.showDetails && (
-					<p className="shipping-label__item-tracking">
-						{ translate( 'Tracking #: {{trackingLink/}}', {
-							components: { trackingLink: <TrackingLink { ...label } /> },
-						} ) }
-					</p>
+					<React.Fragment>
+						<p className="shipping-label__item-tracking">
+							{ translate( 'Tracking #: {{trackingLink/}}', {
+								components: {
+									trackingLink: (
+										<TrackingLink tracking={ label.tracking } carrierId={ label.carrierId } />
+									),
+								},
+							} ) }
+						</p>
+						<ClipboardButton compact onCopy={ this.handleOnCopyClick } text={ label.tracking }>
+							{ translate( 'Copy to clipboard' ) }
+						</ClipboardButton>
+					</React.Fragment>
```

Comment:
this doesn't matter much, but often we do this too

```js
import React, { Fragment } from 'react';

…

<Fragment>
	…
</Fragment>
```

Reply:
Thanks, that's neater :)